src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -70,12 +72,20 @@
                                           :data accounts}
                       :description-props {:text (i18n/label :t/all-addresses)}}
                      {:title             (i18n/label :t/for-airdrops)
-                      :on-press          not-implemented/alert
+                      :on-press          #(rf/dispatch
+                                           [:show-bottom-sheet
+                                            {:content (fn [] [airdrop-addresses/view
```

Comment:
Because you are passing the data on a `on-press`, this effectively makes everything in that view non-reactive. I think you should just initialise the view and subscribe within that view, so that if things changes while the view is renderered, things change accordingly.